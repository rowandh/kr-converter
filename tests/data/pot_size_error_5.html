
<iframe width="0" height="0" name="hfrm"></iframe>
<script language="javascript">
	function goProc(mode,uid,infotype,money){
		form = document.form1;
		form.target="hfrm";
		if (money < 1) {
			alert("1보다 큰금액만 입력하실수 있습니다");
			return;
		}
		form.infotype.value=infotype;
		form.money.value=money;
		form.user_id.value=uid;
		form.action="userinmoney_proc.asp";
		
			if(confirm(uid + "에게 " + money + "원 하자처리 하시겠습니까?")){
				form.submit();
			}		

	}
</script>

<style>
		body {background:#f5f5f5;}
		* {padding:0; margin:0;}
		input:focus,
		select:focus {
			outline:none;
		}
		.page-layout {
			height:100%;
			
		}
		
		.page-layout .bar-search {
			display:flex;
			justify-content: space-between;
			align-items: center;
			flex-wrap: wrap;
			gap:0.875rem;
			
			padding:0 0.875rem 0.875rem;
			background:#2e2f39;
		    border-bottom: 1px solid #e6e6f2;
		    box-shadow: 0 1px 4px 0 rgba(82,63,105,.15);
		}
		
		.page-layout .bar-search .title {
			padding-top:0.875rem;
			font-family:'Malgun Gothic';
			color:#c5ff88;
			font-size:1.425rem;
			font-weight:bolder;
			letter-spacing:-2px;
		}
		
		.page-layout .bar-search .title > span {
			font-size:1rem;
			letter-spacing:0;
		}
		
		.page-layout .bar-search > div:nth-child(2) {
			flex-grow:1;
			
			display:inline-flex;
    		flex-wrap: wrap;
    		justify-content: flex-end;
			gap:0.875rem;
			
			padding-top:0.875rem;
			border-top:1px solid #60931e;
		}
		
		.page-layout .bar-search > div:nth-child(2) span {
		    line-height:2rem;
		    color: #fff;
		    font-weight:bold;
		}
		
		.page-layout .bar-search > div:nth-child(2) select,
		.page-layout .bar-search > div:nth-child(2) input {
		    height: calc(2em + 0.8rem + 2px);
		    padding:0 1rem;
		    font-size: 1rem;
		    font-weight: 400;
		    height:2rem;
		    line-height:2rem;
		    color: #ccc;
		    background-color: #141529;
		    background-clip: padding-box;
		    border: 1px solid #141529;
		    border-radius: 0.25rem;
		    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
		}
		
		.page-layout .bar-search > div:nth-child(2) input.btn-serarch {
			color:#fff;
			font-weight:bold;
			background:#73a730 ;
			border:1px solid #629322;
		}
		
		
		.table-area {
			border:1px solid #e6e6f2;
			border-radius:1rem;
			overflow-x:auto;
			background:#fff;
			box-shadow:3px 3px 5px #e7e7e7;
			margin:1rem;
		}
		
		.table-area th {
			padding: 0 0.5rem;
			font-size:1rem;
			height:46px;
			font-weight: 600;
    		color: #2e2f39;
			background:#fff;
			border-bottom:2px solid #e6e6f2;
		}
		
		.table-area th a {
			color:#cf0b0b;
			text-decoration:none;
		}
		
		.table-area td {
			font-size:14px;
			padding:0.425rem 0.75rem;
			color:#71748d;
			white-space:nowrap;
			background:#fff;
			border-bottom:1px solid #e6e6f2;
		}
		/*
		.table-area tr:nth-child(2n) td {
			background:#f4f4f5;
			border-top:1px solid #e6e6f2;
			border-bottom:1px solid #e6e6f2;
		}
		*/
		.table-area th,
		.table-area td {
			border-right:1px solid #e6e6f2;
		}
		
		.table-area table td a {
			display:inline-block;
			
			padding:0 1rem 0 1.425rem;
			line-height:1rem;
			color:#5e8d21;
			font-size:14px;
			font-weight:bold;
			text-decoration:none;
			border-radius:0.312rem;
			background:url('../images/icon-move.png') left 50% no-repeat;
		}
		
		.table-area table td a:hover {
			filter:hue-rotate(275deg);
		}
		
		.table-area table th.double-table,
		.table-area table td.double-table {
			padding:0;
		}
		
		
		.table-area table th.double-table th {
			padding:0 0.75rem;
			border-bottom:none;
		}
		
		.table-area table td.double-table td {
			line-height:1.5rem;
			border:none;
			background:none;
		}
		
		.table-area table td.double-table tr td+td {
			border-left:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr+tr td {
			border-top:1px solid #e6e6f2;
		}
		
		.table-area table td.double-table tr:nth-child(2n) td {
			background:#f9f9f9;
		}
		
		.btn-history-back {
			padding:0 1rem;
			font-size:1rem;
			height:2.625rem;
			line-height:2.625rem;
			color:#fff;
			font-weight:bold; 
			background:#37a730; 
			border-radius:0.25rem; 
			border:1px solid #629322; 
			box-shadow:1px 1px 0 #629322;
			cursor:pointer;
		}
		
		.btn-history-back:hover {
			filter:brightness(1.1);
			border:1px solid #37a730; 
			box-shadow:1px 1px 0 #37a730;
		}
		
	</style>
	
	<div class="page-layout">
		<div class="bar-search">
			<div class="title">게임기록보기</div>
			<div>
                <div>
					<input type="button" value="닫기" onclick="self.close()" class="btn-serarch" />
                </div>	
			</div>
		</div>
		
		<div class="table-area">
			<table cellspacing="0" width="100%"  border="0" >
				<tr>
                          <th width="" height="34"><p align="center">라운드ID</p></th>
                          <th width="" height="34"><p align="center">시각</p></th>
                          <th width="" height="34"><p align="center">게임 종류</p></th>
                          <th width="" height="34"><p align="center">승자(족보)</p></th>
					<th width="" height="34"><p align="center">이긴금액</p></th>
					<th class="double-table">
						상세정보
					</th>
				</tr>




                            <tr bgcolor="white">
								<td width="56" height="26"><p align="center">&nbsp;15-2-91931708</p></td>
                                <td height="26" ><p align="center">&nbsp;2025-02-08 오전 3:16:42</p></td>
                                <td ><p align="center">&nbsp;홀덤</p></td>
                                <td ><p align="center">&nbsp;<b>StarJ (2 트리플)</b></p></td>
<!--								<td ><p align="center">&nbsp;42615</p></td>   -->
								<td ><p align="center">&nbsp;42,615</p></td>
								<td class="double-table">
									
									<table cellspacing="0" width="100%"  border="0">										
										<colgroup>
											<col width="200" />
											<col width="*" />
											<col width="160" />
											<col width="200" />
										</colgroup>
										<tr>
											<th height="34" style="word-break:break-all"><p align="center">참가자</p></th>
											<th height="34" style="word-break:break-all"><p align="center">족보</p></th>
											<th height="34" style="word-break:break-all"><p align="center">변동 금액</p></th>
											<th height="34" style="word-break:break-all"><p align="center">남은 잔액</p></th>
										</tr>
										
										


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>Bbnak</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>◆</font>7 ♠3 [7 탑]  (다이) 
<br>* 시작 : [StageNo:91931708] [Credit:116,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[bbnak]
<br>* 앤티: -1,000원(115,000원)
<br>* 홀 카드딜: <font color='red'>◆</font>7(19) ♠3(2) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(<font color='red'>◆</font>7 ♠3)]
<br>* 베팅: 다이 [0](115,000원) - 베팅순서: [0][7]
[439ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7♠3) C (♣3♠5<font color='red'>◆</font>2)
<br>* 커뮤니티 카드 딜: H(<font color='red'>◆</font>7♠3) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 종료: WinMoney[0원] Credit[115,000원]
<br>* 결과: 패배 [족보:7 탑] [카드:<font color='red'>◆</font>7 ♠3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">115,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>ripdifunt</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣Q <font color='red'>♥</font>9 [Q 원 페어]  (다이) 
<br>* 시작 : [StageNo:91931708] [Credit:210,258원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[ripdifunt]
<br>* 앤티: -1,000원(209,258원)
<br>* 홀 카드딜: ♣Q(50) <font color='red'>♥</font>9(34) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(♣Q <font color='red'>♥</font>9)]
<br>* 베팅: 콜 -4,000원(205,258원) - 베팅순서: [0][6]
[1636ms]
<br>* 커뮤니티 카드 딜: H(♣Q<font color='red'>♥</font>9) C (♣3♠5<font color='red'>◆</font>2)
<br>* 턴 시작: [플랍(1)] [족보:Q 탑(♣Q <font color='red'>♥</font>9 ♠5 ♣3 <font color='red'>◆</font>2)]
<br>* 베팅: 콜 -1,000원(204,258원) - 베팅순서: [1][4]
[1636ms]
<br>* 커뮤니티 카드 딜: H(♣Q<font color='red'>♥</font>9) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 턴 시작: [턴(2)] [족보:Q 원 페어(<font color='red'>◆</font>Q ♣Q <font color='red'>♥</font>9 ♠5 ♣3)]
<br>* 베팅: [하프] (16,000원) Credit(188,258원) - 베팅순서: [2][3]
[1684ms]
<br>* 접속종료:[1][997]
<br>* 방나감(접속종료)

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-22,000</font></p></td>
											<td style="word-break:break-all"><p align="right">188,258</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>kljdnx</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>7 <font color='red'>◆</font>4 ♣3 ♠5 <font color='red'>◆</font>2 <font color='red'>◆</font>Q [Q 탑]  (다이) (Small Blind)
<br>* 시작 : [StageNo:91931708] [Credit:96,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[kljdnx]
<br>* 앤티: -1,000원(95,000원)
<br>* 홀 카드딜: <font color='red'>♥</font>7(32) <font color='red'>◆</font>4(16) [7 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:7 탑(<font color='red'>♥</font>7 <font color='red'>◆</font>4)]
<br>* 베팅: [블라인드:SMALL] [금액:1,000원] [Creadit:94,000원]
<br>* 베팅: 콜 -3,000원(91,000원) - 베팅순서: [0][8]
[1043ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>7<font color='red'>◆</font>4) C (♣3♠5<font color='red'>◆</font>2)
<br>* 턴 시작: [플랍(1)] [족보:7 탑(<font color='red'>♥</font>7 ♠5 <font color='red'>◆</font>4 ♣3 <font color='red'>◆</font>2)]
<br>* 베팅: 체크 -0원(91,000원) - 베팅순서: [1][1]
[1893ms]
<br>* 베팅: 콜 -1,000원(90,000원) - 베팅순서: [1][5]
[774ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>7<font color='red'>◆</font>4) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 턴 시작: [턴(2)] [족보:Q 탑(<font color='red'>◆</font>Q <font color='red'>♥</font>7 ♠5 <font color='red'>◆</font>4 ♣3)]
<br>* 베팅: 체크 -0원(90,000원) - 베팅순서: [2][1]
[1469ms]
<br>* 베팅: 다이 [0](90,000원) - 베팅순서: [2][4]
[1159ms]
<br>* 종료: WinMoney[0원] Credit[90,000원]
<br>* 결과: 패배 [족보:Q 탑] [카드:<font color='red'>◆</font>Q <font color='red'>♥</font>7 ♠5 <font color='red'>◆</font>4 ♣3] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-6,000</font></p></td>
											<td style="word-break:break-all"><p align="right">90,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>wepoipweo</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠Q ♣6 [Q 탑]  (다이) 
<br>* 시작 : [StageNo:91931708] [Credit:512,967원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:200]
<br>* NICKNAME:[wepoipweo]
<br>* 앤티: -1,000원(511,967원)
<br>* 홀 카드딜: ♠Q(11) ♣6(44) [Q 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:Q 탑(♠Q ♣6)]
<br>* 베팅: 다이 [0](511,967원) - 베팅순서: [0][5]
[211ms]
<br>* 커뮤니티 카드 딜: H(♠Q♣6) C (♣3♠5<font color='red'>◆</font>2)
<br>* 커뮤니티 카드 딜: H(♠Q♣6) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 종료: WinMoney[0원] Credit[511,967원]
<br>* 결과: 패배 [족보:Q 탑] [카드:♠Q ♣6] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">511,967</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>mnbkhjtglg</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>J <font color='red'>♥</font>10 [J 탑]  (다이) 
<br>* 시작 : [StageNo:91931708] [Credit:381,181원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[mnbkhjtglg]
<br>* 앤티: -1,000원(380,181원)
<br>* 홀 카드딜: <font color='red'>♥</font>J(36) <font color='red'>♥</font>10(35) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(<font color='red'>♥</font>J <font color='red'>♥</font>10)]
<br>* 베팅: 다이 [0](380,181원) - 베팅순서: [0][3]
[3333ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J<font color='red'>♥</font>10) C (♣3♠5<font color='red'>◆</font>2)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>J<font color='red'>♥</font>10) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 종료: WinMoney[0원] Credit[380,181원]
<br>* 결과: 패배 [족보:J 탑] [카드:<font color='red'>♥</font>J <font color='red'>♥</font>10] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">380,181</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>rkdcjf</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣8 ♠6 [8 탑]  (다이) 
<br>* 시작 : [StageNo:91931708] [Credit:95,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[rkdcjf]
<br>* 앤티: -1,000원(94,000원)
<br>* 홀 카드딜: ♣8(46) ♠6(5) [8 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:8 탑(♣8 ♠6)]
<br>* 베팅: 다이 [0](94,000원) - 베팅순서: [0][1]
[2584ms]
<br>* 커뮤니티 카드 딜: H(♣8♠6) C (♣3♠5<font color='red'>◆</font>2)
<br>* 커뮤니티 카드 딜: H(♣8♠6) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 종료: WinMoney[0원] Credit[94,000원]
<br>* 결과: 패배 [족보:8 탑] [카드:♣8 ♠6] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">94,000</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>iywffhkvx</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♠J ♣9 ♣3 ♠5 <font color='red'>◆</font>2 [J 탑]  (다이) (Big Blind)
<br>* 시작 : [StageNo:91931708] [Credit:106,603원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[iywffhkvx]
<br>* 앤티: -1,000원(105,603원)
<br>* 홀 카드딜: ♠J(10) ♣9(47) [J 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:J 탑(♠J ♣9)]
<br>* 베팅: [블라인드:BIG] [금액:1,000원] [Creadit:104,603원]
<br>* 베팅: 콜 -3,000원(101,603원) - 베팅순서: [0][9]
[2052ms]
<br>* 커뮤니티 카드 딜: H(♠J♣9) C (♣3♠5<font color='red'>◆</font>2)
<br>* 턴 시작: [플랍(1)] [족보:J 탑(♠J ♣9 ♠5 ♣3 <font color='red'>◆</font>2)]
<br>* 베팅: 체크 -0원(101,603원) - 베팅순서: [1][2]
[316ms]
<br>* 베팅: 다이 [0](101,603원) - 베팅순서: [1][6]
[212ms]
<br>* 커뮤니티 카드 딜: H(♠J♣9) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 종료: WinMoney[0원] Credit[101,603원]
<br>* 결과: 패배 [족보:J 탑] [카드:♠J ♣9 ♠5 ♣3 <font color='red'>◆</font>2] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-5,000</font></p></td>
											<td style="word-break:break-all"><p align="right">101,603</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>gtelu940hs</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											<font color='red'>♥</font>A ♠10 [A 탑]  (다이) 
<br>* 시작 : [StageNo:91931708] [Credit:342,958원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:300]
<br>* NICKNAME:[gtelu940hs]
<br>* 앤티: -1,000원(341,958원)
<br>* 홀 카드딜: <font color='red'>♥</font>A(26) ♠10(9) [A 탑]
<br>* 턴 시작: [프리플랍(0)] [족보:A 탑(<font color='red'>♥</font>A ♠10)]
<br>* 베팅: 다이 [0](341,958원) - 베팅순서: [0][4]
[1434ms]
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>A♠10) C (♣3♠5<font color='red'>◆</font>2)
<br>* 커뮤니티 카드 딜: H(<font color='red'>♥</font>A♠10) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 종료: WinMoney[0원] Credit[341,958원]
<br>* 결과: 패배 [족보:A 탑] [카드:<font color='red'>♥</font>A ♠10] - 기권

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>-1,000</font></p></td>
											<td style="word-break:break-all"><p align="right">341,958</p></td>
											
										</tr>


									
										<tr bgcolor="white">
											<td style="word-break:break-all" bgcolor="white"><p align="center"><b><font color=>StarJ</font></b></p></td>
											
											<td style="word-break:break-all"><p align="left">
											♣2 ♠2 ♣3 ♠5 <font color='red'>◆</font>2 <font color='red'>◆</font>Q [2 트리플]  
<br>* 시작 : [StageNo:91931708] [Credit:97,000원] [SB:1,000원] [BB:1,000원] [MBI:100,000원] [CBIR:100]
<br>* NICKNAME:[starj]
<br>* 앤티: -1,000원(96,000원)
<br>* 홀 카드딜: ♣2(40) ♠2(1) [2 원 페어]
<br>* 턴 시작: [프리플랍(0)] [족보:2 원 페어(♣2 ♠2)]
<br>* 베팅: [쿼터] (4,000원) Credit(92,000원) - 베팅순서: [0][2]
[2421ms]
<br>* 커뮤니티 카드 딜: H(♣2♠2) C (♣3♠5<font color='red'>◆</font>2)
<br>* 턴 시작: [플랍(1)] [족보:2 트리플(<font color='red'>◆</font>2 ♣2 ♠2 ♠5 ♣3)]
<br>* 베팅: [베팅] (1,000원) Credit(91,000원) - 베팅순서: [1][3]
[2701ms]
<br>* 커뮤니티 카드 딜: H(♣2♠2) C (♣3♠5<font color='red'>◆</font>2) (<font color='red'>◆</font>Q)
<br>* 턴 시작: [턴(2)] [족보:2 트리플(<font color='red'>◆</font>2 ♣2 ♠2 <font color='red'>◆</font>Q ♠5)]
<br>* 베팅: [베팅] (1,000원) Credit(90,000원) - 베팅순서: [2][2]
[180ms]
<br>* 종료: WinMoney[42,615원] Credit[132,615원]
<br>* 결과: 승리 [족보:2 트리플] [카드:<font color='red'>◆</font>2 ♣2 ♠2 <font color='red'>◆</font>Q ♠5] - 기권승

											</p></td>
											
											<td style="word-break:break-all"><p align="right"><font color='red'>35,615</font></p></td>
											<td style="word-break:break-all"><p align="right">132,615</p></td>
											
										</tr>

									 </table>
								</td>
                            </tr>






						
                        </table>
                        
                        <table cellpadding="0" cellspacing="0" border="0" width="100%">
                            <tr>
                                <td width="100%" align="center" style="padding:1rem 0; background:#fdfdfd;">
                                	<input type="button" value="닫기" onclick="self.close()" class="btn-history-back">
                                </td>
                            </tr>
                        </table>
                        
		</div>
                        
	</div>




<form name="form1" method="Post">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="logno" value="15-2-91931708">
<input type="hidden" name="regdate_f" value="">
<input type="hidden" name="user_id">
<input type="hidden" name="userid">
<input type="hidden" name="rebateid">
<input type="hidden" name="topid">
<input type="hidden" name="infotype">
<input type="hidden" name="money">
</form>

<form name="form3" method="post">
<input type="hidden" name="idx">
<input type="hidden" name="p_logno">
<input type="hidden" name="s_scroll" value="">
<input type="hidden" name="s_key" value="">
<input type="hidden" name="page" value="">

<input type="hidden" name="s_regdate_y1" value="">
<input type="hidden" name="s_regdate_m1" value="">
<input type="hidden" name="s_regdate_d1" value="">
<input type="hidden" name="s_regdate_h1" value="">
<input type="hidden" name="game_id" value="">
<input type="hidden" name="s_regdate" value="">
</form>


