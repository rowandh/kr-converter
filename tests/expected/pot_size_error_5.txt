PokerStars Hand #15291931708:  Hold'em No Limit ($1000/$1000) - 2025/02/08 03:16:42 KST
Table 'Table 1' 9-max Seat #7 is the button
Seat 1: rkdcjf ($95000 in chips)
Seat 2: StarJ ($97000 in chips)
Seat 3: mnbkhjtglg ($381181 in chips)
Seat 4: gtelu940hs ($342958 in chips)
Seat 5: wepoipweo ($512967 in chips)
Seat 6: ripdifunt ($210258 in chips)
Seat 7: Bbnak ($116000 in chips)
Seat 8: kljdnx ($96000 in chips)
Seat 9: iywffhkvx ($106603 in chips)
rkdcjf: posts the ante $1000
StarJ: posts the ante $1000
mnbkhjtglg: posts the ante $1000
gtelu940hs: posts the ante $1000
wepoipweo: posts the ante $1000
ripdifunt: posts the ante $1000
Bbnak: posts the ante $1000
kljdnx: posts the ante $1000
iywffhkvx: posts the ante $1000
kljdnx: posts small blind $1000
iywffhkvx: posts big blind $1000
*** HOLE CARDS ***
rkdcjf: folds
StarJ: raises $3000 to $4000
mnbkhjtglg: folds
gtelu940hs: folds
wepoipweo: folds
ripdifunt: calls $4000
Bbnak: folds
kljdnx: calls $3000
iywffhkvx: calls $3000
*** FLOP *** [3c 5s 2d]
kljdnx: checks
iywffhkvx: checks
StarJ: bets $1000
ripdifunt: calls $1000
kljdnx: calls $1000
iywffhkvx: folds
*** TURN *** [3c 5s 2d] [Qd]
kljdnx: checks
StarJ: bets $1000
ripdifunt: raises $15000 to $16000
kljdnx: folds
StarJ collected $42615 from pot
*** SUMMARY ***
Total pot $45000 | Rake $2385
Board [3c 5s 2d Qd]
Seat 1: rkdcjf mucked [8c 6s]
Seat 2: StarJ showed [2c 2s] and won ($42615)
Seat 3: mnbkhjtglg mucked [Jh Th]
Seat 4: gtelu940hs mucked [Ah Ts]
Seat 5: wepoipweo mucked [Qs 6c]
Seat 6: ripdifunt mucked [Qc 9h]
Seat 7: Bbnak mucked [7d 3s]
Seat 8: kljdnx mucked [7h 4d]
Seat 9: iywffhkvx mucked [Js 9c]